doc/pipeable-operators.md

```diff
@@ -265,7 +255,7 @@ import { map, filter, reduce } from 'rxjs/operators';
 becomes:
 
 ```ts
-import { map } from 'rxjs/operators/map';
-import { filter } from 'rxjs/operators/filter';
-import { reduce } from 'rxjs/operators/reduce';
+import { map } from 'rxjs/internal/operators/map';
+import { filter } from 'rxjs/internal/operators/filter';
+import { reduce } from 'rxjs/internal/operators/reduce';
```

Comment:
These should all be imported from `rxjs/operators` ... importing anything under `rxjs/internal/**` is a no-no.

Reply:
I think that's intentional because the paragraph above it says:

> #### No Control over Build Process
> If you have no control over your build process (or are unable to upgrade to Webpack 3+), the above solution will not work. Therefore importing from rxjs/operators will likely make your application bundle larger. However, there's still a way you can use pipeable operators. You will have to use deep imports, similar to how you import prior to version 5.5 and pipeable operators:

Reply:
Hmmm... I'm not sure about this. People may want to use the `rxjs-compat` library instead. I'd hate to give people the idea that using anything under `internal` was anything but a pathway to upgrade pain whenever v7 comes up. :thinking:

Reply:
I'm pretty against recommending people reach into `rxjs/internal/` as it should be used as a signal that the API is not public and subject to change without notice. If we want to expose the individual operators as paths it should be in a non-internal path.

I do sympathize with the desire to import these directly for people who don't have access to tree shaking, e.g. react-native folks don't get shaking yet if you use Metro bundler. 

> People may want to use the rxjs-compat library instead. 

I don't believe rxjs-compat exposes pipeable operators with individual paths, does it?

Reply:
> I don't believe rxjs-compat exposes pipeable operators through individual paths, does it?

It does, in fact... it basically re-enables imports from RxJS 5 locations:  `rxjs/operators/map` for example.

Reply:
@jayphelps is absolutely right, anything under internal can change without notice. So we can't recommend anyone uses that stuff.

Reply:
Hmm oh sorry I recalled that rxjs/operators/map wasn’t pipeable but expected `this`. I’ll look again when I get a sec. 👍👍